pyspecProcScripts/convert_to_power.py

```diff
@@ -0,0 +1,148 @@
+import pyspecdata as psd
+import pyspecProcScripts as prscr
+import h5py
+import matplotlib.pyplot as plt
+import numpy as np
+from Instruments.logobj import logobj
+import datetime
+import re
+
+
+def convert_to_power(s, filename, exp_type, fl=None, node_name="log"):
+    """Generate power axis for ODNP/E(p)/FIR experiments, converts instrument
+    power log to useable axis
+    Parameters
+    ===========
+    s: nddata
+        This is an nddata whose "indirect"
+        axis is a structured array.  This structured array gives the
+        `start_times` and `stop_times` (as fields) for each datapoint
+        along the indirect axis.
+    filename : str
+        Name of the hdf file that contains the power log.
+    exp_type : str
+        The "experiment type" used to search for the file.
+    fl: figlist_var()
+
+    Returns
+    =======
+    values: numpy.array
+    s: nddata
+        (modified in-place)
+        When we're done, the axis coordinates for "indirect" are a (not
+        structured) array containing the average power recorded on the
+        log between each start_time and stop_time.
+        The axis coordinate errors are the standard deviations of the
+        same.
+    """
+    my_filename = psd.search_filename(
```

Comment:
This is where I was previously using `find_file` where it would complain as the 'log' node is not nddata, Here, using `search_filename` and then reading the file below has been working fine